src/platform/qt/scripting/ScriptingView.cpp

```diff
@@ -105,7 +110,9 @@ void ScriptingView::updateMRU() {
 	m_ui.mru->clear();
 	for (const auto& fname : m_mruFiles) {
 		m_ui.mru->addAction(fname, [this, fname]() {
-			m_controller->loadFile(fname);
+			if(m_controller->loadFile(fname)) {
```

Comment:
```suggestion
			if (m_controller->loadFile(fname)) {
```

I had wondered why this didn't work, but didn't look into it. Thanks for fixing that!